test/packages/good_v3/_dev/test/config.yml

```diff
@@ -0,0 +1,10 @@
+system:
+  parallel: true
+pipeline:
+  parallel: false
+asset:
+  parallel: false
+policy:
+  parallel: false
+static:
+  parallel: false
```

Comment:
Nice! Maybe we can also use this common config to skip all the tests of a kind.

Reply:
Adding a new field `skip` in every kind of tests ? Something like this? 
```yaml
static:
  parallel: false
  skip: true
```

IIUC that would avoid to add the skip field in every test config file?

Reply:
Yes, this would be useful for example to disable all system tests if there is some issue with the service deployment.

Reply:
Added `skip`. :+1: 

I also renamed `parallel` to be `sequential`, not sure if that would be better or not.

Thinking in `elastic-package` , if `sequential` is not defined it means is `false` by default, and therefore tests could be run in parallel.

WDYT ?

Reply:
I think I prefer `parallel` to `sequential`, but not a strong opinion, as you prefer.